src/assessments/adaptable-content/assessment.tsx

```diff
@@ -14,9 +14,9 @@ import { Reflow } from './test-steps/reflow';
 import { ResizeText } from './test-steps/resize-text';
 import { TextSpacing } from './test-steps/text-spacing';
 
-const key = 'textLegibility';
-const title = 'Text legibility';
-const { guidance } = content.textLegibility;
+const key = 'adaptableContent';
```

Comment:
Be careful about backwards compatibility; it will probably be necessary to avoid changing whatever keys names appear in IndexedDB data for storing assessment results, otherwise any users with an assessment in progress at the time they receive the update with this change will end up with broken assessments.

Reply:
So should I keep the key as text legibility? Or is there another way for me to ensure backwards compatibility?

Reply:
If I'm remembering correctly, the `key` is the important thing to maintain. The `storeDataKey` at L59 also makes me suspicious based on its name, but not so much based on the code using it.

I'd recommend testing it out to see what happens (and to validate any updates you might try making here):

* Check out master
* Start an assessment and fill in a few results for Text Legibility
* Open up the dev tools on the background page and look through the stored IndexedDB data for the assessment
* In particular, Ctrl+F through the whole contents for the terms "Text Legibility" or "textLegibility"; it's likely that anywhere it appears as a key for a dictionary (as opposed to a value) will need to be preserved
* Close chrome, checkout your feature branch, rebuild, open chrome
* Continue the assessment in progress and check IndexedDB again; see if the in-progress results persisted appropriately across the upgrade

Note that I'm making an assumption here that we would want in-progress results to persist across the rename; if you haven't already, it would be good to double check that assumption with @ferBonnin. We do sometimes intentionally invalidate in-progress results if there is a significant enough change to the requirements/text. It's also possible that we don't think users would care either way; if that's true, the consistency might be more valuable than a few users having to redo their Text Legibility results for one assessment.

An alternative would be to rekey the requirement for consistency and update the `InitialAssessmentStoreDataGenerator` to understand how to convert from the old key name to the new one when loading persisted data. I think that's probably a bigger hammer than this change warrants, but it's an option.

Reply:
Did this a couple of times to be sure (building master first, then my feature branch and then vice versa), and can confirm you're right! It will remove the assessment data for text-legibility/adaptable-content if you rename the key! @ferBonnin let me know if overriding in progress results for that one assessment (the data for other assessments remains in tact)

Reply:
Great catch Dan! @JGibson2019 we don't want our users to loose their data from the text legibility/adaptable content test.

Reply:
Ok tested that everything else is fine if we leave the key as `text-legibility`. Out of curiosity, will there ever be a point where we rename the key? Are there any other (maybe unintended) consequences of having the key name not match the assessment name? @dbjorge

Reply:
If we were to make a different change in the future that dramatically changed most/all of the requirements, such that we would want to intentionally invalidate any existing results, it would make sense to change the key then. That sort of change is fairly rare; I would expect it to be unlikely to happen within the product's lifetime.

I think the main negative consequence of leaving the key as-is permanently are that it might make debugging/logging a little confusing in the future, but as long as the old key is the only place we retain `textLegibility`, I don't think it's a big deal; it's not hard to search the codebase and find what it really refers to.

If we decided we really wanted to change it for consistency's sake, you could do it by changing the key here but separately updating the point where we read in previously-persisted data with new logic to convert from the old key to the new one. That conversion logic could potentially be ripped out a few releases later. I think if there were bigger disadvantages to the inconsistency it would make sense to take this approach - it's not particularly difficult or dangerous, just adds a bit of dev cost to write the transformation logic and test it. I don't feel like it's worth the time in this case, since I don't think the inconsistency causes any real problems for us.

It would probably be a good idea to leave a comment next to the `key` definition explaining that the inconsistency is intentional for backcompat purposes